packages/images/lib/plugin.js

```diff
@@ -37,6 +47,28 @@ export default class ImageElementPlugin extends BasePlugin {
     },
   };
 
+  shouldAvoidProxy (method, url, body) {
+    // if it looks like we have an image element in the url (as a route
+    // parameter), never proxy. Just look for our image element prefix in allowed
+    // positions (either after an 'element' or 'screenshot' path segment), and
+    // ensure the prefix is followed by something
+    if (IMG_EL_URL_RE.test(url)) {
+      return true;
+    }
+
+
+    // also if it looks like we have an image element in the request body (as
+    // a JSON parameter), never proxy. Basically check against a regexp of the
+    // json string of the body, where we know what the form of an image element
+    // must be
+    const stringBody = JSON.stringify(body);
```

Comment:
is body always defined?

Reply:
it appears to be; i just moved this code from basedriver so it should be working exactly as it was before